src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsVelesTracking.java

```diff
@@ -27,34 +34,101 @@
 
   public static final String VELES_TRACKING_PRICE_PARAMETER = "bidder_8";
 
+  private void testVeles(String urlFragment, Map<String, String> positionsAndPrices) {
+    AdsBaseObject pageObject = new AdsBaseObject(driver, urlBuilder.getUrlForPage(
+        new Page("project43", urlFragment)
+    ));
+
+    Set<HarEntry> entries = findEntriesByUrlPart(
+        pageObject,
+        VELES_TRACKING_REQUEST_PART,
+        positionsAndPrices
+    );
+
+    Assertion.assertTrue(
+        wasVelesTrackedIn(entries, positionsAndPrices),
+        "Veles should be tracked only in " + positionsAndPrices
+    );
+  }
+
   @NetworkTrafficDump()
-  @Test(
-      groups = "AdsTrackingVeles",
-      dataProviderClass = AdsDataProvider.class,
-      dataProvider = "adsVelesTracking"
-  )
-  public void adsTrackingVelesTracked(final Page page, final Map<String, String> positionsAndPrices) {
+  @Test(groups = {"AdsTrackingVelesOasis", "AdsTrackingVelesTrackedForBothSlots"})
+  public void adsTrackingVelesTrackedForBothSlots() {
     networkTrafficInterceptor.startIntercepting();
-    AdsBaseObject pageObject = new AdsBaseObject(driver, urlBuilder.getUrlForPage(page));
+    testVeles(
+        "SyntheticTests/RTB/Prebid.js/Veles?" + AdsVeles.TURN_ON_QUERY_PARAM,
+        buildPositionsAndPrices("20.00", "20.00")
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "20.00" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)